### ISSUE TITLE
Document exported from MS Word not compliant

### ISSUE THREAD

**Issue by alexreg** [veraPDF/veraPDF-library#1133]
I am exporting an MS Word using the "accessible" (i.e., PDF-A) option. I then validate it with PDF/A, but it turns out it's not compliant. Is this the fault of MS Word's service or a veraPDF mistake?

[Output](https://gist.github.com/5c3b91260b52eb0ce4d0be86baefacd8)

**Comment by bdoubrov** [comment]
By looking at the validation report, I'd say the wrong options are chosen when exporting the document to PDF. Note that "accessible" corresponds to PDF/UA standard, which is different from PDF/A.

Feel free to attach also the resulting PDF: I would then be able to reconfirm the above.

**Comment by alexreg** [comment]
Thanks for the response. Unfortunately no options are offered by Word. What I just tried is to use Ghostscript to convert the document to a PDF/A compliant one.

```sh
gs -dBATCH -dNOPAUSE -dPDFSETTINGS=/prepress -dNOOUTERSAVE -sColorConversionStrategy=UseDeviceIndependentColor -sProcessColorModel=DeviceCMYK -sDEVICE=pdfwrite -dCompatibilityLevel=1.7 -dPDFA -sPDFACompatibilityPolicy=3 -sOutputFile=food-pda.pdf foo.pdf
```

I think this is correct? (Unfortunately, there is still an issue with CID fonts when I then embed this into a LaTeX document using `pdfpages` and generate a PDF from that that using XeTeX, but I suspect the problem is with `pdfpages`. I include the output of veraPDF on the resulting XeTeX-generated PDF in case you are curious, in any case:

```xml
?xml version="1.0" encoding="utf-8"?>
<report>
  <buildInformation>
    <releaseDetails id="core" version="1.16.1" buildDate="2020-05-12T00:43:00+01:00"></releaseDetails>
    <releaseDetails id="validation-model" version="1.16.1" buildDate="2020-05-12T00:46:00+01:00"></releaseDetails>
    <releaseDetails id="gui" version="1.16.1" buildDate="2020-05-12T00:59:00+01:00"></releaseDetails>
  </buildInformation>
  <jobs>
    <job>
      <item size="1391811">
        <name>/Users/alex/Documents/University of Bristol/MA PoP/Philosophy of Physics/Essay/build/essay.pdf</name>
      </item>
      <validationReport profileName="PDF/A-3U validation profile" statement="PDF file is not compliant with Validation Profile requirements." isCompliant="false">
        <details passedRules="125" failedRules="1" passedChecks="21907" failedChecks="5">
          <rule specification="ISO 19005-3:2012" clause="6.2.11.4" testNumber="4" status="failed" passedChecks="0" failedChecks="5">
            <description>If the FontDescriptor dictionary of an embedded CID font contains a CIDSet stream, then it shall identify all CIDs which are present in the font program,
                        regardless of whether a CID in the font is referenced or used by the PDF or not.</description>
            <object>PDCIDFont</object>
            <test>fontFile_size == 0 || fontName.search(/[A-Z]{6}\+/) != 0 || CIDSet_size == 0 || cidSetListsAllGlyphs == true</test>
            <check status="failed">
              <context>root/document[0]/pages[1](14 0 obj PDPage)/contentStream[0]/operators[11]/xObject[0](56 0 obj PDXForm)/contentStream[0](56 0 obj PDContentStream)/operators[1082]/font[0](EXFMZX+Calibri)/DescendantFonts[0](EXFMZX+Calibri)</context>
            </check>
            <check status="failed">
              <context>root/document[0]/pages[1](14 0 obj PDPage)/contentStream[0]/operators[11]/xObject[0](56 0 obj PDXForm)/contentStream[0](56 0 obj PDContentStream)/operators[978]/font[0](IVVNTW+Calibri)/DescendantFonts[0](IVVNTW+Calibri)</context>
            </check>
            <check status="failed">
              <context>root/document[0]/pages[1](14 0 obj PDPage)/contentStream[0]/operators[11]/xObject[0](56 0 obj PDXForm)/contentStream[0](56 0 obj PDContentStream)/operators[969]/font[0](HBPNKF+Calibri-Light)/DescendantFonts[0](HBPNKF+Calibri-Light)</context>
            </check>
            <check status="failed">
              <context>root/document[0]/pages[1](14 0 obj PDPage)/contentStream[0]/operators[11]/xObject[0](56 0 obj PDXForm)/contentStream[0](56 0 obj PDContentStream)/operators[900]/font[0](IVVNTW+Calibri)/DescendantFonts[0](IVVNTW+Calibri)</context>
            </check>
            <check status="failed">
              <context>root/document[0]/pages[1](14 0 obj PDPage)/contentStream[0]/operators[11]/xObject[0](56 0 obj PDXForm)/contentStream[0](56 0 obj PDContentStream)/operators[665]/font[0](JMXUSJ+Calibri-Light)/DescendantFonts[0](JMXUSJ+Calibri-Light)</context>
            </check>
          </rule>
        </details>
      </validationReport>
      <duration start="1611346056868" finish="1611346057995">00:00:01.127</duration>
    </job>
  </jobs>
  <batchSummary totalJobs="1" failedToParse="0" encrypted="0">
    <validationReports compliant="0" nonCompliant="1" failedJobs="0">1</validationReports>
    <featureReports failedJobs="0">0</featureReports>
    <repairReports failedJobs="0">0</repairReports>
    <duration start="1611346056723" finish="1611346059644">00:00:02.921</duration>
  </batchSummary>
</report>
```

(Sorry, it's not so easy to share the PDF right now, as it contains some personal information, but I could perhaps create an MWE in the future with a bit of effort.)

**Comment by alexreg** [comment]
Any updates?

**Comment by bdoubrov** [comment]
Indeed, I see only a single issue related to the CIDSet entry in the font descriptor. We had a related issue https://github.com/veraPDF/veraPDF-library/issues/1112 fixed a while ago, which is not included yet in the latest release. 

So, I recommend you to retest your file with the latest dev build taken from this folder: https://software.verapdf.org/develop/2.0

**Comment by alexreg** [comment]
Thanks for that. Indeed, veraPDF now happily validates my PDF document!

P.S. Using the installer is no problem for me, but I have been trying to create a Homebrew (package manager) formula for installing on macOS, so if you or one of your fellow maintainers can see my issue #1128 when you get a moment, that would be super.

**Comment by alexreg** [comment]
Sorry, I'm not quite sure how it worked once, but trying it again, it seems the same error is still produced... odd.

**Comment by bdoubrov** [comment]
I'm not sure, how we can help here without having access to your PDF documents. And yes, we'll try fixing issue #1128 as well

**Comment by alexreg** [comment]
That's great, cheers.

As for this issue: you're right. I put some effort into creating a MWE now, so [here you go](https://github.com/veraPDF/veraPDF-library/files/5923237/bar.pdf).

**Comment by bdoubrov** [comment]
Fix released in v1.20

**Comment by alexreg** [comment]
Thanks!
